src/core/fft-common.hpp

```diff
@@ -291,7 +291,8 @@ void fft_unpack_block(double *in, double *out, int start[3], int size[3],
                       int dim[3], int element);
 
 /** Debug function to print global fft mesh.
-    Print a globaly distributed mesh contained in data. Element size is element.
+    Print a globally distributed mesh contained in data. Element size is
+ element.
```

Comment:
Extra spaces are also needed here and in some other places.

Reply:
Is there a .cpp or .hpp file I could use as reference to correctly format Doxygen comments in the future?

Reply:
Not really. We were pretty negligient about Doxygen comments. The official documentation is pretty good though: https://www.stack.nl/~dimitri/doxygen/manual/docblocks.html.